lib/consumer.js

```diff
@@ -71,7 +71,8 @@ function Consumer(opts) {
    * @desc A private key for funding the channel. An alternative implementation could
    * provide a list of unspent outputs and the keys needed to sign the outputs
    */
-  this.fundingKey = opts.fundingKey || new PrivateKey();
+  $.checkArgument(opts.fundingKey instanceof PrivateKey, 'fundingKey is expected to be a PrivateKey');
+  this.fundingKey = opts.fundingKey;
```

Comment:
This changes the behavior. Previously, if no fundingKey was supplied, it would create a new one for you.